tutorials/nbconverted/general_usage.py

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Pycytominer General Usage Walkthrough!
+# 
+# Welcome to this walkthrough where we will guide you through the process of extracting single cell morphology features using the [`pycytominer`](https://github.com/cytomining/pycytominer) API.
+# 
+# For this walkthrough, we will be working with the NF1-Schwann cell morphology dataset. 
+# If you would like the more information about this dataset, you can refer to this [repository](https://github.com/WayScience/Benchmarking_NF1_data)
+# 
+# From the mentioned repo, we specifically used this [dataset](https://github.com/WayScience/Benchmarking_NF1_data/tree/main/4_processing_features/data/Plate2/CellProfiler) and the associated [metadata](https://github.com/WayScience/Benchmarking_NF1_data/tree/main/3_extracting_features/metadata) to generate the walkthrough. 
+# 
+# 
+# Let's get started with the walkthrough below!
+
+# In[17]:
+
+
+import pathlib
+
+import pandas as pd
+
+# pycytominer imports
+from pycytominer.cyto_utils.cells import SingleCells
+from pycytominer import annotate, normalize, feature_select
+
+# ignore warnings
```

Comment:
Recommend adding a reason why you are ignoring the warnings. Since this is a tutorial, it might be good for someone new to pycytominer to know why the warnings aren't helpful.